CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### BREAKING CHANGES
 
+- **CUMULUS-1212**
+  - `@cumulus/post-to-cmr` will now fail if any granules being processed are missing a metadata file.
```

Comment:
👍

Reply:
Though this should be a new `BREAKING CHANGES` header under `Unreleased`

Reply:
Oh snap. it was right before the new release went out. I'll update